index.html

```diff
@@ -0,0 +1,191 @@
+              <div class="words">
+                <img src="image/Excursion.png" alt="awards" />
+                <a href="#">Awards</a>
+                <p>
+                  There will be awards for the best web developers in the
+                  country.
+                </p>
+              </div>
+              <div class="words">
+                <img src="image/Exhibition.png" alt="Forums" />
+                <a href="#">Forums</a>
+                <p>
+                  There will be a competition during the web developers
+                  conference.
+                </p>
+              </div>
+              <div class="words">
+                <img src="image/Exhibition.png" alt="networks" />
+                <a href="#">Networks</a>
+                <p>
+                  Web Devs can network properly among themselves during the
+                  conference.
+                </p>
+              </div>
+              <div class="words">
+                <img src="image/Excursion.png" alt="Workshop" />
+                <a href="#">Workshop</a>
+                <p>
+                  There will be a Workshop during the web Dev conference in
+                  Lagos.
+                </p>
+              </div>
+            </div>
+            <a href="#" class="follow">View the Whole Program</a>
+            <button type="submit" class="submit-btn">
+              Join Conference 2023
+            </button>
+          </div>
+        </div>
+      </section>
+
+      <!--Program section-->
+
+      <section class="speakers" id="featured-speakers">
+        <h2 class="open">Featured Speakers</h2>
+        <div class="speakers-container"></div>
+        <div class="attention" id="new-comers"></div>
+      </section>
+
+      <!--supporters section-->
+
+      <section id="suporters">
+        <div class="support-container">
+          <div class="partners">
+            <h2>Partners</h2>
+            <div class="people"></div>
+            <div class="individuals">
+              <img src="image/airbnb.png" alt="airbnb" class="partner-img" />
+              <img src="image/amazon.png" alt="amazon" class="partner-img" />
+              <img src="image/coke.png" alt="coca-cola" class="partner-img" />
+              <img
+                src="image/facebook.png"
+                alt="facebook"
+                class="partner-img"
+              />
+              <img src="image/mozilla.png" alt="mozilla" class="partner-img" />
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <!--footer section-->
+
+      <section class="footer" id="footer-section">
+        <div class="materials">
+          <div class="copyright">
+            <img src="image/logoimg.jpg" alt="logo" />
+            <p>Web Developers conference</p>
+          </div>
+          <div class="year">
+            <p>&copy; 2023 Web dev conference. All rights Reserved</p>
+          </div>
+        </div>
+      </section>
+    </div>
```

Comment:
Let's work on the design to match the [Figma design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) given for this project.

The desktop version looks good 👍, however, we need to adjust the design for mobile.

Figma design:
<img width="467" alt="image" src="https://github.com/Megagig/Capstone-Project/assets/31897434/9ba7e3ae-6147-40a4-bac9-67d857e727ff">

Your design: (kindly remove the partners section on mobile)
<img width="497" alt="image" src="https://github.com/Megagig/Capstone-Project/assets/31897434/06f95895-e4f2-42fa-b7c3-b04874b171cd">